rabbitmq/plan.sh

```diff
@@ -62,3 +62,7 @@ do_build() {
 do_check() {
   make tests
 }
+
+do_setup_environment() {
+  set_runtime_env RABBITMQ_LOG_BASE "${pkg_svc_var_path}/logs"
```

Comment:
I'm curious why this is set in the plan and not in ./config/env with the other env vars

Reply:
💯 Agree. Why did I do that? Pushed a new commit to move to `config/env`